src/RulesEngine/RulesCache.cs

```diff
@@ -16,20 +16,20 @@ internal class RulesCache
         private ConcurrentDictionary<string, (IDictionary<string, RuleFunc<RuleResultTree>>, Int64)> _compileRules = new ConcurrentDictionary<string,  (IDictionary<string, RuleFunc<RuleResultTree>>, Int64)>();
 
         /// <summary>The workflow rules</summary>
-        private ConcurrentDictionary<string, (WorkflowRules, Int64)> _workflowRules = new ConcurrentDictionary<string, (WorkflowRules, Int64)>();
+        private ConcurrentDictionary<string, (Workflow, Int64)> _Workflows = new ConcurrentDictionary<string, (Workflow, Int64)>();
```

Comment:
can we ensure we are using lower CamelCase for privates and variables? It looks like many variables got changed to uppercase

Reply:
Fixed - was caught in bad search and replace - lesson learned: use Match Case. :)